readthedocs/projects/tasks.py

```diff
@@ -723,6 +781,12 @@ def move_files(version_pk, hostname, html=False, localmedia=False, search=False,
                 type_='epub', version_slug=version.slug, include_file=False)
             Syncer.copy(from_path, to_path, host=hostname)
 
+    if linguist:
+        from_path = version.project.artifact_path(
+            version=version.slug, type_='linguist')
+        target = version.project.rtd_build_path(version.slug)
+        Syncer.copy(from_path, target, host=hostname)
```

Comment:
Do we actually need to do this? I thought we weren't copying the linguist files to the webs?

Reply:
I also think the `target` might be overwriting the built HTML if synced with `rsync --delete`, which would be bad (https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/builds/syncers.py#L69) :)

Reply:
Ya, we shouldn't need it.